core/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -1446,6 +1446,14 @@ public long getLastWriteNanos() {
     public abstract void deactivateThrottling();
 
     /**
+     * Marks operations in the translog from the specified local checkpoint as completed. This is used to restore the state of the local
```

Comment:
Okay, I pushed an explanation to the place in index shard where we use the method. 😄